packages/suite/src/components/suite/PrerequisitesGuide/DeviceDisconnectRequired.tsx

```diff
@@ -3,7 +3,6 @@ import { Translation } from 'src/components/suite/Translation';
 
 export const DeviceDisconnectRequired = () => (
     <TroubleshootingTips
-        initiallyIsOpen={true}
```

Comment:
to keep prev behaviour intact - we should set this to false here

Reply:
but maybe we wanted to open the modal automatically - semantically this seems to be the intention 🤷